fastcgi/Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM ubuntu:24.04
+
+RUN apt-get update && apt-get install -y git vim apache2 apache2-dev make gcc \
+    php-fpm php-zip php-dom php-xml php-gd php-sqlite3 php-mbstring php-curl
```

Comment:
should use a fixed php version number and have that in the directory name as well?